src/ui/qgsexpressionselectiondialogbase.ui

```diff
@@ -85,22 +85,22 @@
   </layout>
   <action name="mActionSelect">
    <property name="text">
-    <string>Select features</string>
```

Comment:
Can you update the i18n folder too? This will break translations.

Reply:
If you are sure that it's picked up like that by lrelease all good for me.

Reply:
I'm far from sure -- I have zero experience with translations

Reply:
A simple `sed` over the source strings in `i18n/*.ts` might bring us to the safe side here. If not, I'd rather have translation work and accept imperfect capitalization in the English version.

Reply:
Oops I just realised I misread where your comment was - I thought it was on the select by expression change

Reply:
Wait - it is. Ignore me

Reply:
So I do see all these strings in i18n/*.ts : e.g.

``qgis_hi.ts:        <source>Filter Current Selection</source>```

Does that mean it's safe to backport?

Reply:
I found it in `qgis_de.ts` (without capitalization, obviously)

Reply:
Alright. I'll drop.